Add hands-on tutorial for go-dqlite (#11)
canonical/dqlite-docs: tutorials/get-started-with-dqlite-in-go.md

> This step-by-step guide walks you through the setup of an Dqlite example application. 
> By the end of this tutorial you will have a fully-functional Dqlite cluster deployed on your machine and you'll be able to interact with it through a HTTP client.

Reviewer: ```suggestion
This step-by-step guide walks you through the setup of a Dqlite example application. 
By the end of this tutorial you will have a fully-functional Dqlite cluster deployed on your machine and you'll be able to interact with it through an HTTP client.
```

## What you’ll need:

> * A working station, e.g. a laptop

Reviewer 2: "working station" is not correct I think

Author: Fixed

* Some familiarity with Go

## What you’ll do:

> * [Setup your development environment](tbd)

Reviewer: ```suggestion
* [Set up your development environment](tbd)
```

* [Create an example application](tbd)

## Next steps:

> In the next step we'll [setup your development environment](tbd).

Reviewer: ```suggestion
In the next step we'll [set up your development environment](tbd).
```
Just as an explanation: It's "to set up" as a verb, but "a setup" as a noun. So you will "set something up", and then you'll have a "setup".

Author: Thanks for the explanation - learned something today :)

But there is a lot more to explore:

| IF YOU ARE WONDERING... | VISIT...                                                            |
|-------------------------|---------------------------------------------------------------------|
| "How do I...?           | [How-to docs](tbd)             |
| "What is ...?           | [Reference docs](tbd)    |

> | "Why...?", "So what?"   | [Explanation docs](tbd) |

Reviewer: This is a good idea, but unfortunately it won't work with the current setup of the docs. There aren't any pages to link to here, since How-to/Reference/Explanation are just headings for now and not linked to a page.
So I would leave out the table for now.
